js/scripts.js

```diff
@@ -1 +1,7 @@
- 
\ No newline at end of file
+ var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.";
+ var animal = "Zielone słonie";
+ var animalUpperCased = animal.toUpperCase();
+ var textAfter = text.replace('Papugi', animalUpperCased);
+ console.log(textAfter.length/2);
+ var textEnd = textAfter.slice(0, 45);
```

Comment:
Jak zmieni się długość zmiennej `text` to przestanie ona dzielić równo na pół - masz jakiś pomysł jak rozwiązać ten problem?